frontend/packages/client/src/components/Community/ProposalThresholdEditor/ProposalThresholdEditor.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useWebContext } from 'contexts/Web3';
+import { ActionButton, WrapperResponsive } from 'components';
+import Checkbox from 'components/common/Checkbox';
+import Input from 'components/common/Input';
+import { yupResolver } from '@hookform/resolvers/yup';
+import isEqual from 'lodash/isEqual';
+import networks from 'networks';
+import { Schema } from './FormConfig';
+
+const networkConfig = networks[process.env.REACT_APP_FLOW_ENV];
+
+const defaultVaules = {
+  contractAddress: networkConfig.flowAddress.contractAddr,
+  contractName: networkConfig.flowAddress.contractName,
+  storagePath: networkConfig.flowAddress.storagePath,
+  proposalThreshold: '0',
+};
+
+const checkFieldValues = ({
+  contractAddress,
+  contractName,
+  storagePath,
+  proposalThreshold,
+  onlyAuthorsToSubmitProposals,
+}) => {
+  return (
+    onlyAuthorsToSubmitProposals === true &&
+    isEqual(defaultVaules, {
+      contractAddress,
+      contractName,
+      storagePath,
+      proposalThreshold,
+    })
+  );
+};
```

Comment:
if checked on onlyAllowAuthorsToSubmit and did not complete any of the fields they will be filled automatically by this [code](https://github.com/DapperCollectives/CAST/pull/357/files#diff-7e73186cafb85fc0ea763743f363017fd2382cb443c26648ae27eb3fc7dc085cR147-R155) so did this to make the form empty same as when they were creating the community. Otherwise it will be populated with the information sent to the backend and could be confusing since they did not entered any information when creating the community.